x/bank/legacy/v036/types.go

```diff
@@ -12,9 +12,3 @@ type (
 		Supply sdk.Coins `json:"supply" yaml:"supply"`
```

Comment:
Please note that some files, such as this one, are not deleted. That's because the 0.39->0.40 migrations references a struct in v038, which references a struct in v036 etc.

These files can probably be removed when we deprecate the 0.39 migration too, in a future release.

Reply:
should we add a comment about that?